users/tests/test_api_authentication.py

```diff
@@ -90,61 +90,66 @@ def test_refresh_token_valid_data_response_success(
 
 @pytest.mark.django_db()
 def test_refresh_token_invalid_data_response_failed(
-    client: Client, user: User, login_url: str, token_refresh_url: str, user_credentials: dict[str, str]
+    client: Client, user: User, api_login_url: str, token_refresh_url: str, user_credentials: dict[str, str]
 ) -> None:
-    client.post(login_url, user_credentials)
+    client.post(api_login_url, user_credentials)
     response = client.post(token_refresh_url, {"refresh": "123"})
     assert response.status_code == 401
     assert "access" not in response.data
     assert "refresh" not in response.data
 
 
 @pytest.mark.django_db()
-def test_logout_valid_data_response_success(
-    client: Client, user: User, login_url: str, logout_url: str, user_credentials: dict[str, str]
+def test_api_logout_valid_data_response_success(
+    client: Client, user: User, api_login_url: str, api_logout_url: str, user_credentials: dict[str, str]
 ) -> None:
-    response = client.post(login_url, user_credentials)
-    refresh_token = response.data["refresh"]
-    response = client.post(logout_url, {"refresh": refresh_token})
+    login_response = client.post(api_login_url, user_credentials)
```

Comment:
Great! 
Now please create a fixture for user logged in via API.
You can also modify django client to send JWT in headers like so:
`client.credentials(HTTP_AUTHORIZATION='Bearer ' + token)`
This will save us couple lines in every test.